packages/harmony/src/components/radio-group/RadioGroup.tsx

```diff
@@ -40,9 +36,9 @@ export const RadioButtonGroup = (props: RadioButtonGroupProps) => {
   )
   return (
     <RadioGroupContext.Provider value={{ name, onChange: handleChange, value }}>
-      <div {...divProps} role='radiogroup'>
+      <Flex role='radiogroup' direction='column' {...other}>
```

Comment:
Nice. Maybe we should support `asChild` instead though, and pass our `role='radiogroup'` in by default to the child?

Reply:
Added to circle back on this when radio get's implemented the right way, just wanted to avoid too many changes in this migration, considering the radiogroup rendered a div originally, love this idea though makes a lot of sense